caddyhttp/staticfiles/fileserver.go

```diff
@@ -40,6 +40,15 @@ func (fs FileServer) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, err
 	return fs.serveFile(w, r, r.URL.Path)
 }
 
+func isNotDirErr(err error) bool {
+	perr, ok := err.(*os.PathError)
+	if !ok {
+		return false
+	}
+
+	return perr.Err.Error() == "not a directory"
+}
+
```

Comment:
Can we move this lower in the file to keep the serveFile and ServeHTTP methods close together?